api/tools/src/main/java/org/pmiops/workbench/tools/WorkspaceExportRow.java

```diff
@@ -62,7 +122,7 @@ public void setProjectId(String projectId) {
   }
 
   @CsvBindByName(column = "Workspace Project ID")
-  @CsvBindByPosition(position = 4)
+  @CsvBindByPosition(position = 9)
```

Comment:
One option is to just write an array of JSON objects and use an off-the-shelf json->csv converter.